crawler/helpers/exceptions.py

```diff
@@ -0,0 +1,3 @@
+class CherrypickerDataError(Exception):
+    def __init__(self, message):
+        self.message = message
```

Comment:
There's already a file crawler/exceptions.py - but I expect you're aware and this either needed to be under the helpers folder or that's good practice.